.github/workflows/codesee-arch-diagram.yml

```
@@ -5,77 +5,16 @@ on:
   pull_request_target:
     types: [opened, synchronize, reopened]
 
-name: CodeSee Map
+name: CodeSee
+
+permissions: read-all
```

```suggestion
permissions:
  actions: read
  contents: read
  pull-requests: read
```
Let's see if this is enough.